packages/template-retail-react-app/app/components/experience/image-with-text/index.jsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React, {useEffect, useState} from 'react'
+import PropTypes from 'prop-types'
+import {Image, Box, Text, Link} from '@chakra-ui/react'
+import DOMPurify from 'dompurify'
+
+/**
+ * Image with text component
+ *
+ * @param ITCLink Image Link.
+ * @param ITCText Text Below Image.
+ * @param image Image.
+ * @param heading Text Overlay.
+ * @param alt The image alt text shown by the component.
+ * @returns {JSX.Element}
+ */
+const ImageWithText = ({ITCLink, ITCText, image, heading, alt}) => {
+    const [sanitizedHtml, setSainitizedHtml] = useState(null)
+
+    useEffect(() => {
+        const purify = DOMPurify(window)
+        const cleanHeading = purify.sanitize(heading)
+        const cleanITCText = purify.sanitize(ITCText)
+        setSainitizedHtml({heading: cleanHeading, ITCText: cleanITCText})
+    }, [])
+
+    return (
+        <div className={'image-with-text'}>
```

Comment:
What are you thoughts on using native html elements? I'm wondering if we should only be using them when deem necessary, otherwise we should use Chakra component. Only reason I bring this up woud be for consistency and promote chakra usage.

Reply:
In the PWA Kit context makes sense to use Chakra UI components when possible over the native HTML elements. I made the change.